packages/shared/ReactFeatureFlags.js

```diff
@@ -166,6 +166,9 @@ export const enableCustomElementPropertySupport = __NEXT_MAJOR__;
 // request for certain browsers.
 export const enableFilterEmptyStringAttributesDOM = __NEXT_MAJOR__;
 
+// Use this flag to remove legacy APIs.
+export const enableRemoveLegacyAPIs = __NEXT_MAJOR__;
```

Comment:
Calling it this because I want to overload it with removing the other legacy APIs like `.render`, since turning this off and leaving the others on is unsupported since it breaks act.